src/app/pages/ConsensusDashboardPage/LearningMaterials.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import Grid from '@mui/material/Unstable_Grid2'
+import { LearningMaterialsCard } from 'app/components/LearningMaterialsCard'
+import { LearningSection } from '../../components/LearningMaterialsCard/LearningSection'
+import { docs } from '../../utils/externalLinks'
+
+export const LearningMaterials: FC = () => {
+  const { t } = useTranslation()
+
+  return (
+    <LearningMaterialsCard>
```

Comment:
Oh, this wont use ParatimeDashboardPage/LearningMaterials.tsx? https://github.com/oasisprotocol/explorer/blob/92b1259966107035153a0e4cfd5810684d49b770/src/app/pages/ParatimeDashboardPage/LearningMaterials.tsx#L127 type is inaccurate then and only works until Layer.consensus becomes always enabled in ENABLED_LAYERS_FOR_NETWORK

Reply:
added commit that introduce a new type